src/main/scala/dpla/ingestion3/mappers/providers/OrbisCascadeMapping.scala

```diff
@@ -1,127 +1,135 @@
         .map(nameOnlyAgent)
 
-  // done
   override def creator(data: Document[NodeSeq]): Seq[EdmAgent] =
-    extractStrings(metadataRoot(data) \ "creator")
+    extractStrings(data \\ "SourceResource" \ "contributor")
       .map(nameOnlyAgent)
 
-  // done
-  override def date(data: Document[NodeSeq]): Seq[EdmTimeSpan] =
-    extractStrings(metadataRoot(data) \ "date")
-      .map(stringOnlyTimeSpan)
+  // TODO: Verify no label
+  override def date(data: Document[NodeSeq]): ZeroToMany[EdmTimeSpan] = {
+    val dateEarly = extractStrings(data \\ "SourceResource" \ "date" \ "TimeSpan" \ "begin")
+    val dateLate = extractStrings(data \\ "SourceResource" \ "date" \ "TimeSpan" \ "end")
+
+    if(dateEarly.length == dateLate.length) {
+      dateEarly.zip(dateLate).map {
+        case (begin: String, end: String) =>
+          EdmTimeSpan(
+            originalSourceDate = Some(s"$begin-$end"),
+            begin = Some(begin),
+            end = Some(end)
+          )
+      }
+    } else {
+      Seq(emptyEdmTimeSpan)
+    }
+  }
 
   // done
   override def description(data: Document[NodeSeq]): Seq[String] =
-    extractStrings(metadataRoot(data) \ "description")
-
-  // done
-  override def identifier(data: Document[NodeSeq]): Seq[String] =
-    extractStrings(metadataRoot(data) \ "identifier")
-      .filterNot(t => {
-        Try { new URL(t) } match {
-          case Success(_) => true
-          case Failure(_) => false
-        }
-      })
+    extractStrings(data \\ "SourceResource" \ "description")
 
   // done
   override def language(data: Document[NodeSeq]): Seq[SkosConcept] =
-    extractStrings(metadataRoot(data) \ "language")
+    extractStrings(data \\ "SourceResource" \ "language")
       .map(nameOnlyConcept)
 
-  // done
-  override def publisher(data: Document[NodeSeq]): Seq[EdmAgent] =
-    extractStrings(metadataRoot(data) \ "publisher")
-      .map(nameOnlyAgent)
-
   // done
   override def place(data: Document[NodeSeq]): Seq[DplaPlace] =
-    extractStrings(metadataRoot(data) \ "coverage")
+    extractStrings(data \\ "SourceResource" \ "spatial")
       .map(nameOnlyPlace)
 
   // done
   override def subject(data: Document[NodeSeq]): Seq[SkosConcept] =
-    extractStrings(metadataRoot(data) \ "subject")
+    extractStrings(data \\ "SourceResource" \ "subject")
       .map(nameOnlyConcept)
 
   // done
   override def title(data: Document[NodeSeq]): Seq[String] =
-    extractStrings(metadataRoot(data) \ "title")
+    extractStrings(data \\ "SourceResource" \ "title")
 
   // done
   override def `type`(data: Document[NodeSeq]): Seq[String] =
-    extractStrings(metadataRoot(data) \ "type")
+    extractStrings(data \\ "SourceResource" \ "hasType")
 
   // OreAggregation
   override def dplaUri(data: Document[NodeSeq]): ZeroToOne[URI] = mintDplaItemUri(data)
 
-  // TODO FIXME - TBD could be source or could be publisher
+  // FIXME: Need to dereference the URI
```

Comment:
Don't we blow up if we don't have a dataProvider?

Reply:
yes -- this is what I'm asking them to fix and they are opting for a texas style provider lookup table (including text labels is out of scope for remediation).